csp/handlers/handlers.go

```diff
@@ -361,8 +361,8 @@ func (c *CSPHandlers) ConsumedAddressHandler(w http.ResponseWriter, r *http.Requ
 }
 
 // validateMemberNumber checks if the member number is provided
-func validateMemberNumber(memberID string) error {
-	if len(memberID) == 0 {
+func validateMemberNumber(memberNumber string) error {
```

Comment:
this should happen in the other PR. but maybe we can squash both together and name it "replace memberID with memberNumber and participantID"

Reply:
will be separate PRs